lib/utils/showHtmlToast.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+// eslint-disable-next-line import/prefer-default-export
+export function showHtmlToast(messageId, config = { messageType: 'success' }) {
+  this.callout.current.sendCallout({
```

Comment:
I don't think you should use `this`here

Reply:
probably we should have `withCallout` or context that will pass cb with connected callout object

Reply:
I'm not sure if it's so easy with refs. `showToast` uses the same technique, BTW

Reply:
I agree with @aliaksei-chumakou.

Reply:
it's mostly the same what we have (in case context usage), ref will be in closure within callback and `showToast` will be passed with context consumer

HoC is simpler, but we will have callout component in each place (not only at the top level) and  `showToast` will be passed via props